### DISCUSSION TITLE
How to use in php 7 or 8?

### DISCUSSION THREAD

**Discussion by M-J-King** [mysticzhong/docx2html#2] [Q&A] [answered]
How to use in php 7 or 8, maybe it is some incompatible in that env.

include('Docx2Html.php');

$Docx2Html = new Docx2Html('test1');  // test2 test3
$html = $Docx2Html->transferToHtml();
// print_r($html);
$url = $Docx2Html->writeToHtml($html);
print_r($url);

$Docx2Html->deleteTemp();
make song wrong.

**Comment by whisky-rye** [comment, marked as answer]
Yeah, It develop on php 5.7,but ont test on 7 or higher, you can fix on main function.

**Reply by M-J-King** [reply]
ok, I know that.

**Comment by liuzhihang** [comment]
good

**Comment by ghost** [comment]
good
